src/styles/App.css

```
@@ -165,9 +165,9 @@ a:focus {
   height: 100dvh; /* Dynamic viewport height for mobile */
   color: white;
   background-color: #000000;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family:
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
```

Stylelint (17.13.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)